grunt-configs/copy.js

```diff
@@ -31,11 +31,12 @@ module.exports = function (grunt, options) {
                         '**/*'
                     ],
                     dest: options.staticHashDir + 'javascripts/vendor'
-                },
-                {
+                },{
                     expand: true,
-                    cwd: options.requirejsDir,
-                    src: [
+                    cwd: options.isDev ? options.staticSrcDir + 'javascripts' : options.requirejsDir,
```

Comment:
We shouldn't be using `AssetMap`, because we don't need to resolve paths to hashed paths.